src/AlignLoads.cpp

```diff
@@ -43,7 +45,21 @@ class AlignLoads : public IRMutator {
                                  index, load->image, load->param));
     }
 
+    void visit(const Call *op) {
+        bool old_inside_address_of = inside_address_of;
```

Comment:
Rather than track whether inside address_of or not, I think this visitor should just not recurse into address_of calls.